plugins/importexport/users/UserImportExportPlugin.inc.php

```diff
@@ -43,6 +43,39 @@ function executeCLI($scriptName, &$args) {
 	function usage($scriptName) {
 		fatalError('Not implemented.');
 	}
+	/**
+	 * Define the appropriate import filter given the imported XML file path
+	 *
+	 * @param string $xmlFile
+	 *
+	 * @return array Containing the filter and the xmlString of the imported file
+	 */
+	function getImportFilter($xmlFile) {
+		fatalError('Not implemented.');
```

Comment:
Convert here (and above and below) to exceptions.

Reply:
Converted